app/presenters/step_by_step_presenter.rb

```diff
@@ -0,0 +1,33 @@
+class StepByStepPresenter
```

Comment:
I like the idea of a presenter, I'm just concerned that the name is too similar to `StepNavPresenter`  and could cause confusion. Perhaps `StepByStepPagePresenter` so it's named similarly to the model?